contracts/multiswap-contracts/FeeDistributor.sol

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "@openzeppelin/contracts/utils/cryptography/draft-EIP712.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+
+contract FeeDistributor is EIP712, Ownable {
+    using SafeERC20 for IERC20;
+    using ECDSA for bytes32;
+
+    string constant NAME = "FEE_DISTRIBUTOR";
```

Comment:
make publicly readeable. So that offchain sig creation can use these instead of constants